ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
+        static let avatarViewWidth: CGFloat = 70.0
+        static let avatarViewHeight: CGFloat = 70.0
+        static let firstNameLabelMinimumHeight: CGFloat = 12.0
+        static let spacingBetweenAvatarAndFirstLabel: CGFloat = 13.0
+    }
+
+    private var avatarView: AvatarView
+    private var firstNameLabel: UILabel?
+    private var lastNameLabel: UILabel?
+    private var identifierLabel: UILabel?
+
+    /// Initializes the contact view with an avatar view, first name, and last name
+    ///
+    /// - Parameters:
+    ///   - avatarView: The AvatarView that will be displayed within the ContactView
+    ///   - firstName: String that will be the text of the top label
+    ///   - lastName: String that will be the text of the bottom label
+    public init(avatarView: AvatarView, firstName: String, lastName: String) {
+        self.avatarView = avatarView
+        self.firstNameLabel = UILabel()
```

Comment:
>self [](start = 8, length = 4)

no need for "self." unless a name conflict is here

Reply:
Ah I see! Thanks for pointing it out